### PR TITLE
feat(prediction): add popup popover

### LINKED ISSUE #530: [FEAT] Prediction demo: Add popup/popover on "running elements"
Display at least information about the start date, predicted end date, late time.
We can use tippy.js as we do in custom behavior example.
This feature should have been included as part of #528 but
> For now, there is a problem to add popup with tippy. It brokes our CSS for the label of the executed elements.

Do we let user perform actions with button inside the popover? Not for now

### PR DESCRIPTION
**Pull request by csouchet**
**Live environment of examples**

[https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/260-add_popup_popover/examples/index.html](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/260-add_popup_popover/examples/index.html)

https://github.com/process-analytics/bpmn-visualization-examples/assets/4921914/1cd6bae1-2fb6-4181-9751-566082c64c08

Closes #530

### CODE REVIEW

File: demo/prediction/js/prediction-use-case.js [hunk 1]
```diff
@@ -16,20 +16,20 @@ class PredicatedLateUseCase extends UseCase {
...
-        this._style.highlightRunningElementsWithPrediction(this._executionData.runningElementWithPrediction);
+        this._style.highlightRunningElementWithPrediction(this._executionData.runningElementWithPrediction);
         this._style.toggleHighlightRunningElementsWithoutPrediction(this._executionData.runningElementsWithoutPrediction);
-        this._registerInteractions(this._executionData.predictedPaths, this._executionData.runningElementsWithoutPrediction, this._executionData.runningElementWithPrediction);
+        this._registerInteractions(this._executionData.runningElementWithPrediction.id, this._executionData.predictedPaths, this._executionData.runningElementsWithoutPrediction);
     }
 
     _initManagers() {
-        this._style = new PredicatedLateStyle(this._bpmnVisualization.bpmnElementsRegistry);
+        this._style = new PredicatedLateStyle(this._bpmnVisualization.bpmnElementsRegistry, `${this.type}-bpmn-container`);
```
Comment (tbouffard) [inline demo/prediction/js/prediction-use-case.js:25]: **question:** can't we retrieve the id from the UseCase class? we are leaking the internal implementation of UseCase here!

**Review by tbouffard — CHANGES_REQUESTED** [review]
✔️ rendering: very nice, great job ❤️ https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/da46821/demo/prediction/index.html
❌ IMHO we should change the way the tippy/popper assets are loaded (see my comments)
❌ the predicted dates are not releavant for the domain. It is about baking and delivering pizza, so the duration should be minutes (or eventually few hours for late). I cannot be on time and take 2 days 😄 

image

File: demo/prediction/index.html [hunk 2]
```diff
@@ -16,13 +16,18 @@
 <head>
     <meta charset="UTF-8">
     <title>bpmn-visualization - Predictions demo</title>
+
+    <script src="https://unpkg.com/@popperjs/core@2"></script>
+    <script src="https://unpkg.com/tippy.js@6"></script>
```
[the remark this answers is not in the document]
Comment (csouchet) [reply demo/prediction/index.html:21]: I just follow the documentation of tippy: https://atomiks.github.io/tippyjs/v6/getting-started/#2-cdn

File: demo/prediction/js/data.js [hunk 3]
```diff
@@ -12,6 +45,9 @@ class ExecutionData {
 
     _nonPredictedElements;
 
+    /**
+     * format: {{id:string, startDate:string, predictedEnd:string, currentDelay: string, predictedDelay: string}}
+     */
```
[the remark this answers is not in the document]
Comment (csouchet) [reply demo/prediction/js/data.js:50]: Thanks for the tips :wink:

**Pull request merged**